lib/mailer.js

```diff
@@ -9,9 +9,11 @@ var _             = require('lodash'),
 
 var internals = { };
 
-internals.sendLoginEmail = function (config, transporter, log, account, auth, callback) {
+internals.sendLoginEmail = function (config, transporter, log, account, auth, urlType, callback) {
   var subject = config.compiledSubject({account : account, auth : auth});
-  var loginUrl = config.compiledLoginUrl({account : account, auth : auth});
+
+  var compiledLoginUrl = _.template(config.loginUrls[urlType]);
```

Comment:
Pass in the already compiled login templates, otherwise we need to recompile on every request which will slow things down. 

We should also add validation logic so when the plugin is configured there is at least a default login url. I believe the CI build is failing because of the current validation logic, which needs to get updated.